### DISCUSSION TITLE
可以不可以将教材中范例的完整代码提供一下

### DISCUSSION THREAD

**Discussion by willisofsunny** [AmazingAng/WTF-Solidity#316] [General]
可以不可以将教材中范例的完整代码提供一下？？

纯小白（之前没有学会过任何一种编程语言的小白）
依样画葫芦，但一样跑不出跟教材范例一样的效果
一直找不到到底是哪里出错

希望能有范例完整的代码，让小白能跑一样的效果

**Comment by AmazingAng** [comment]
有的，所有代码都开源在这个github repo中
https://github.com/AmazingAng/WTF-Solidity

**Reply by willisofsunny** [reply]
找到了，感谢，突然有有信心继续学下去了～开心
